tsconfig.json

```
@@ -1,36 +1,17 @@
 {
-  "include": [
-    "**/*.ts",
-    "**/*.tsx",
-    "**/.server/**/*.ts",
-    "**/.server/**/*.tsx",
-    "**/.client/**/*.ts",
-    "**/.client/**/*.tsx",
+  "files": [],
+  "references": [
+    { "path": "./tsconfig.node.json" },
+    { "path": "./tsconfig.cloudflare.json" }
   ],
   "compilerOptions": {
-    "lib": ["DOM", "DOM.Iterable", "ES2022"],
-    "types": [
-      "vite/client",
-      "@cloudflare/workers-types/2023-07-01"
-    ],
-    "isolatedModules": true,
-    "esModuleInterop": true,
-    "jsx": "react-jsx",
-    "module": "ESNext",
-    "moduleResolution": "Bundler",
-    "resolveJsonModule": true,
-    "target": "ES2022",
-    "strict": true,
-    "allowJs": true,
+    "checkJs": true,
+    "verbatimModuleSyntax": false,
```

```suggestion
    "verbatimModuleSyntax": true,
```
`verbatimModuleSyntax` はできれば `true` にしたいです.

---

eslintの設定も含めてhttps://github.com/pulsate-dev/caramel/pull/313/commits/28460b295f9de119fce83b189838f7bc1373dff4 でやっておきました